demo/AliceGetsAPhone.md

```diff
@@ -111,38 +104,37 @@ Open a new bash shell, and in a project directory, run:
 git clone https://github.com/bcgov/indy-tails-server.git
 cd indy-tails-server/docker
 ./manage build
-GENESIS_URL=http://test.bcovrin.vonx.io/genesis ./manage start
+./manage start
 ```
 
 This will run the required components for the tails server to function and make a tails server available on port 6543.
 
-This will also automatically start an ngrok server that will expose a public url for your tails server - this is required to support mobile agents.  The docker output will look something like this:
+This will also automatically start an ngrok server that will expose a public url for your tails server - this is required to support mobile agents. The docker output will look something like this:
 
 ```bash
 ngrok-tails-server_1  | t=2020-05-13T22:51:14+0000 lvl=info msg="started tunnel" obj=tunnels name="command_line (http)" addr=http://tails-server:6543 url=http://c5789aa0.ngrok.io
 ngrok-tails-server_1  | t=2020-05-13T22:51:14+0000 lvl=info msg="started tunnel" obj=tunnels name=command_line addr=http://tails-server:6543 url=https://c5789aa0.ngrok.io
 ```
 
-Note the server name in the `url=https://c5789aa0.ngrok.io` parameter (`https://c5789aa0.ngrok.io`) - this is the external url for your tails server.  Make sure you use the `https` url!
-
+Note the server name in the `url=https://c5789aa0.ngrok.io` parameter (`https://c5789aa0.ngrok.io`) - this is the external url for your tails server. Make sure you use the `https` url!
 
 ### Run `faber` With Extra Parameters
 
-If you are running in a *local bash shell*, navigate to [The demo direcory](/demo) and run:
+If you are running in a _local bash shell_, navigate to [The demo direcory](/demo) and run:
 
 ```bash
 TAILS_NETWORK=docker_tails-server LEDGER_URL=http://test.bcovrin.vonx.io ./run_demo faber --revocation --events
 ```
 
 The `TAILS_NETWORK` parameter lets the demo script know how to connect to the tails server (which should be running in a separate shell on the same machine).
 
-If you are running in *Play with Docker*, navigate to [The demo direcory](/demo) and run:
+If you are running in _Play with Docker_, navigate to [The demo direcory](/demo) and run:
 
 ```bash
 PUBLIC_TAILS_URL=https://def456.ngrok.io LEDGER_URL=http://test.bcovrin.vonx.io ./run_demo faber --revocation --events
 ```
 
-The `PUBLIC_TAILS_URL` parameter lets the demo script know how to connect to the tails server.  This can be running in another PWD session, or even on your local machine - the ngrok endpoint is public and will map to the correct location.
+The `PUBLIC_TAILS_URL` parameter lets the demo script know how to connect to the tails server. This can be running in another PWD session, or even on your local machine - the ngrok endpoint is public and will map to the correct location.
```

Comment:
Need to remove use of PUBLIC_TAILS_URL here.  Code that references this env var is no longer used.

Note that TAILS_NETWORK still works based on updates to ngrok_wait.sh

Reply:
I updated the demo to function with either `--tails-server-base-url` or `PUBLIC_TAILS_URL` so this should still work